apps/webapp/src/components/header/header.tsx

```diff
@@ -1,11 +1,21 @@
-import { Navbar } from './navbar';
+import { LayoutLoaderResult } from '../layout.tsx';
 import { Link, useLoaderData } from 'react-router-dom';
-import { PagePath } from '../metadata/paths.ts';
+import { MessageWarningIcon } from '../../icons/message-warning.tsx';
+import { MobileNavMenu } from './mobile-nav-menu.tsx';
+import { Navbar } from './navbar';
+import {
+  NetworksPopover,
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@penumbra-zone/ui';
 import Notifications from './notifications.tsx';
+import { PagePath } from '../metadata/paths.ts';
 import { TabletNavMenu } from './tablet-nav-menu.tsx';
```

Comment:
all of these import changes are just alphabetically ordering them (except for the `Tooltip*` imports)

Reply:
If we don't have a prettier or eslint command to do this automatically, count on this order not being maintained ;)